features/foutafhandeling.feature

```diff
@@ -0,0 +1,292 @@
+# language: nl
+Functionaliteit: Afhandeling van fouten
+
+    Als Consumer van de BAG API
+    Wil ik consistente en gebruiksvriendelijke fout responses conform problem+json
+    Zodat ik in mijn consumer applicatie de fouten eenduidig kan afhandelen.
+
+Rule: verplichte parameter(s) zijn opgegeven
+
+    Scenario: Verplichte parameter is niet opgegeven
+        Als '/adressen/zoek' wordt aangeroepen
+        Dan bevat de response de volgende kenmerken
+        | naam   | waarde                                                                            |
+        | title  | Ten minste één parameter moet worden opgegeven.                                   |
+        | status | 400                                                                               |
+        | code   | paramsRequired                                                                    |
+        | detail | Required request parameter 'zoek' for method parameter type String is not present |
+        En bevat de response geen invalidParams
+
+Rule: alleen gespecificeerde parameters mogen worden gebruikt
```

Comment:
is dit wat jij wilt en verwacht? hadden we dit niet eerder besproken dat onverwachte parameters genegeerd moeten worden? 
Voor [WOZ](https://github.com/VNG-Realisatie/Haal-Centraal-WOZ-bevragen/issues/90#issuecomment-876442528) hebben we besloten onbekende parameters te negeren. Bij BRK worden onbekende parameters ook genegeerd. Waarom wijk je bij BAG daarvan af?

Reply:
Ik wijk niet af. Ik documenteer alleen wat ik tegen kom. Deze feature file is een uitbreiding van een feature file die ik ruim een jaar geleden ter discussie heb aangeboden. Toen werkte het klaarblijkelijk al zo. Ik heb hier niet naar gekeken toen de devs bij WOZ deze issue aankaarten.